pagopa/pn-frontend
# Feat/pn 3695

## Short description
create new logic for deleghe page with tab to change between two type of delegation

## List of changes proposed in this pull request
- insert tab in page
- commented useless part of deleghe page
- create new type of test, where simulate change of tab in page
- fix mobile view

## How to test
click on  deleghe in side bar menu and view title, subtitle and tab for choose which deleghe what do you want to see
